python/mosaic/utils/kepler_magic.py

```diff
@@ -101,8 +101,25 @@ def mosaic_kepler(self, *args):
                 data = data.withColumn(
                     feature_name, lower(conv(col(feature_name), 10, 16))
                 )
+        if feature_type == "bng":
+            data = (data
+                .withColumn(feature_name, grid_boundaryaswkb(feature_name))
+                .withColumn(feature_name, st_geomfromwkb(feature_name))
+                .withColumn(
+                    feature_name,
+                    st_transform(st_setsrid(feature_name, lit(27700)), lit(4326))
```

Comment:
`27700` and `4326` should be two constants somewhere

Reply:
Agreed! will update